hack/make-rules/test-cmd-util.sh

```diff
@@ -2574,6 +2575,22 @@ run_rs_tests() {
   fi
 }
 
+run_daemonset_tests() {
```

Comment:
Why is this tested here? Shouldn't this be a unit test? Is this to test that kubectl doesn't introduce some sort of change? If so please add a comment.